<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8" />
	<link rel="stylesheet" type="text/css" href="../../style.css">
	<title>Teaching Online - Biodiversity Informatics - Introduction</title>
</head>
<body>

<a href="../..">Biodiversity Informatics</a>

<h1>Introduction</h1>

<h2><a name="what"/>What is "biodiversity informatics"?</h2>

According to <a href="https://en.wikipedia.org/wiki/Biodiversity_informatics">Wikipedia</a>:
<blockquote>Biodiversity Informatics is the application of informatics techniques to biodiversity information for improved management, presentation, discovery, exploration and analysis. It typically builds on a foundation of taxonomic, biogeographic, or ecological information stored in digital form, which, with the application of modern computer techniques, can yield new ways to view and analyse existing information, as well as predictive models for information that does not yet exist (see niche modelling). Biodiversity informatics is a relatively young discipline (the term was coined in or around 1992) but has hundreds of practitioners worldwide, including the numerous individuals involved with the design and construction of taxonomic databases. The term "Biodiversity Informatics" is generally used in the broad sense to apply to computerized handling of any biodiversity information; the somewhat broader term "bioinformatics" is often used synonymously with the computerized handling of data in the specialized area of molecular biology.</blockquote>

For some reviews of the field, see:
<ul>
<li>Peterson, A. T., Knapp, S., Guralnick, R., Soberón, J., & Holder, M. T. (2010). The big questions for biodiversity informatics. Systematics and Biodiversity, 8(2), 159–168. <a href="https://doi.org/10.1080/14772001003739369">doi:10.1080/14772001003739369</a></li>
<li>Hardisty, A., Roberts, D. A decadal view of biodiversity informatics: challenges and priorities. BMC Ecol 13, 16 (2013). <a href="https://doi.org/10.1186/1472-6785-13-16">https://doi.org/10.1186/1472-6785-13-16</a></li>
<li>Bingham H, Doudin M, Weatherdon L, Despot-Belmonte K, Wetzel F, Groom Q, Lewis E, Regan E, Appeltans W, Güntsch A, Mergen P, Agosti D, Penev L, Hoffmann A, Saarenmaa H, Geller G, Kim K, Kim H, Archambeau A, Häuser C, Schmeller D, Geijzendorffer I, García Camacho A, Guerra C, Robertson T, Runnel V, Valland N, Martin C (2017) The Biodiversity Informatics Landscape: Elements, Connections and Opportunities. Research Ideas and Outcomes 3: e14059. <a href="https://doi.org/10.3897/rio.3.e14059">https://doi.org/10.3897/rio.3.e14059</a></li>
<li>Parr C.S., Thessen A.E. (2018) Biodiversity Informatics. In: Recknagel F., Michener W. (eds) Ecological Informatics. Springer, Cham <a href="https://doi.org/10.1007/978-3-319-59928-1_17">https://doi.org/10.1007/978-3-319-59928-1_17</a></li>
<li>Guralnick, R., & Hill, A. (2009). Biodiversity informatics: automated approaches for documenting global biodiversity patterns and processes. Bioinformatics, 25(4), 421–428. <a href="https://doi.org/10.1093/bioinformatics/btn659">doi:10.1093/bioinformatics/btn659</a></li>
</ul>

<h2>What is this course about?</h2>

This course provides an opinionated survey of a range of topics in biodiversity informatics, with an emphasis on data discovery and visualisation. Rather than focus on a particular set of questions, or a particular technology (e.g., R) the course ranges widely over a range of topics and aims to give you a sense of the diversity of relevant data, and the varied ways to explore and analyse that data.

<h2><a name="lecture"/>Lecture</h2>

<h3>Notes on lecture</h3>

Below are some notes and updates on things discussed in the lecture.

<h4>Linking</h4>

<uL>
<li>The sequence used in the lecture is <a href="http://www.ncbi.nlm.nih.gov/nucleotide/DQ176312.1" target="_new">DQ176312</a>. Since I made the original slides GenBank has updated the sequence to include the proper name of the ant. You can see the original version of the sequence taxon name using the history tool, see <a href="http://www.ncbi.nlm.nih.gov/nucleotide/DQ176312.1?report=girevhist" target="_new">http://www.ncbi.nlm.nih.gov/nucleotide/DQ176312.1?report=girevhist</a>.</li>

<li>iSpecies is at <a href="http://ispecies.herokuapp.com/?q=Dermophis" target="_new">http://ispecies.herokuapp.com</a>. </li>
</ul>

<h4>Geography</h4>

<ul>
<li>This is the video of a geophylogeny on Google Earth.<br/>

<video width="400" controls="controls">
  <source src="movies/ge.mov" type="video/quicktime" />
  Your browser does not support the video tag.
</video>
<br/><a href="movies/ge.mov">Download</a></li>
</ul>

<h4>Taxonomy</h4>

<ul>
<li>The description of <i>Microgale jenkinsae</i> is available in <a href="http://biostor.org/reference/81349" target="_new">BioStor</a></li>

<li>Touch screen displaying a large classification:<br/>

<video width="400" controls="controls">
  <source src="movies/perceptive.mov" type="video/quicktime" />
  Your browser does not support the video tag.
</video>
<br/>
<a href="movies/perceptive.mov">Download</a></li>

<li>For more examples see <a href="http://iphylo.blogspot.co.uk/2012/09/touching-tree-of-life.html"  target="_new">Touching the tree of life</a> and <a href="http://www.onezoom.org" target="_new">OneZoom</a>.</li>
</ul>


<h2>Exercises</h2>

Below are some activities that we will do during this session. If this course is being taught online we will make use of Mentimeter for many of these exercises.


<h3>Introducing yourself</h3>


<ul>
<li>What things are you interested in?</li>
<li>What skills do you have?</li>
<li>What computer(s) do you use?</li>
</ul>


<h3>How do you find information on a species you are interested in?</h3>

We will create a list of sites/databases/etc. that we can use to find out information on a species (Menti).

Add your results to a shared spreadsheet at <a href="http://bit.ly/2ON8ldi">http://bit.ly/2ON8ldi</a>.



<h3>Finding out about one species</h3>

<p>One basic challenge is to find out what we know about a species. Pick an organism and find out the following:
<ol>
<li>What does it look like?</li>
<li>Were does it live?</li>
<li>What is it's evolutionary history (e.g., closest living relative</li>
<li>Where was it's original description? </li>
<li>Has it been sequenced?</li>
<li>How big is it?</li>
<li>What does it eat?</li>
<li>What parasites does it have? (or if it's a parasite, what are its hosts?)</li>
</ol>
</p>
<p>Possible organisms (or try your own favourites):
<ul>
<li><i>Eleutherodactylus philipi</i></li>
<li><i>Kiwa puravida</i></li>
<li><i>Parasitodiplogaster laevigata</i></li>
<li><i>Alpheus lottini</i></li>
<li><i>Isthmohyla rivularis</i></li>
</ul>

Add your results to a shared spreadsheet at <a href="http://bit.ly/3qyHBuB">http://bit.ly/3qyHBuB</a>.

<h3>Encyclopedia of Life (EOL)</h3>

<blockquote>Comparative biology, crossing the digital divide, has begun a still largely unheralded revolution: the exploration and analysis of biodiversity at a vastly accelerated pace. Its momentum will return systematics from its long sojourn at the margin and back into the mainstream of science. Its principal achievement will be a single-portal electronic encyclopedia of life.  <br />- <b>E. O. Wilson</b> <a href="https://doi.org/10.1016/S0169-5347(02)00040-X">https://doi.org/10.1016/S0169-5347(02)00040-X</a></blockquote>

<iframe width="560" height="315" src="https://www.youtube.com/embed/e-txR1WSPBs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

<p>Inspired by the late E. O. Wilson the <a href="https://eol.org">Encyclopedia of Life</a> aims to be a "one-stop-shop for biodiversity information. Using a species that you are interested (or one from the list above), how well do you think it meets this goal? How does it compare to simple mashups such as <a href="https://ispecies.herokuapp.com/?q=Dermophis">http://ispecies.herokuapp.com</a>?</p>

<h3>The challenge of getting data from web sites</h3>

<p>Web sites may have lots of interesting data, but in a format intended to look nice to people. This can make it hard to extract the underlying data if you want to do soemthing else with that data.</p>

For example below is the <a href="https://www.flickr.com/groups/encyclopedia_of_life/">
Encyclopedia of Life Images</a> group on Flickr (a photo sharing site that predates Instagram). If you wanted to make your own map how would you get this data?

<p>
<img width="80%" src="images/flickr.png">
</p>

<p>Most web pages are designed to be viewed by people not computers, for example <a href="http://en.wikipedia.org/wiki/Mantidactylus" target="_new">http://en.wikipedia.org/wiki/Mantidactylus</a>. </p>

<h4>Viewing web sites as a machine</h4>

<table>
<thead><tr><th>Data type</th><th>URL</th></tr></thead>
<tbody>
<tr><td>HTML</td><td><a href="../../services/inspector.php?url=http://en.wikipedia.org/wiki/Mantidactylus" target="_new">http://en.wikipedia.org/wiki/Mantidactylus</a></td></tr>

<tr><td>CSV</td><td><a href="../../services/inspector.php?url=http%3A%2F%2Fdbpedia.org%2Fsparql%3Fdefault-graph-uri%3Dhttp%253A%252F%252Fdbpedia.org%26query%3DDESCRIBE%2B%253Chttp%3A%2F%2Fdbpedia.org%2Fresource%2FMantidactylus%253E%26format%3Dtext%252Fcsv" target="_new">http://dbpedia.org/sparql?default-graph-uri=http%3A%2F%2Fdbpedia.org&query=DESCRIBE+%3Chttp://dbpedia.org/resource/Mantidactylus%3E&format=text%2Fcsv</a></td></tr>

<tr><td>JSON</td><td><a href="../../services/inspector.php?url=http://dbpedia.org/data/Mantidactylus.json" target="_new">http://dbpedia.org/data/Mantidactylus.json</a></td></tr>

<tr><td>XML</td><td><a href="../../services/inspector.php?url=http://dbpedia.org/data/Mantidactylus.rdf" target="_new">http://dbpedia.org/data/Mantidactylus.rdf</a></td></tr>

</table>

<h3>Using APIs to make something new</h3>

<p>If a web site has an API then we can get the data and do something new and different with that data. For example, Flickr has an RSS feed. This provides the underlying data in a form that computers can use.</p>

<ul>
<li><a href="leafletrss.html">RSS feed on a map</a></li>
</ul>


<h3>Using APIs to make something new</h3>

<p>If we can access the data then we can start to build completely new tools. For example, <a href="https://biorss.herokuapp.com/" target="_new">BioRSS</a> takes RSS feeds from multiple sources (such as scientific journals) and creates a web site that lists the most recent species discoveries.</p>

<p>
<img width="80%" src="images/biorss.png">
</p>

<h2><a name="citizen"/>Creating data</h2>
<p>
We can be more than passive consumers of data, we can contribute data, either  via data archives (such as <a href="https://figshare.org">FigShare</a>),
or via citizen science projects such as <a href="https://inaturalist.org"  target="_new">iNaturalist</a>.
</p>

<p>
<img width="80%" src="images/inat.png">
</p>


<p>Download the iNaturist app (available for both iOS and Android) and explore its functionality.</p>

<p>Note that community-based tools such as iNaturalist depend on the community being well-behaved, for an example of where thiungs can get messy see <a href="https://taxacom.markmail.org/search/?q=#query:+page:1+mid:etko34ivvj43rftg+state:results">[Taxacom] iNaturalist and the dangers of community ID sites!</a>.</p>

<h2><a name="ai"/>AI</h2>

<p>Since the end of 2022 there has been a lot of interest in AI tools such as <a href="https://chat.openai.com"  target="_new">ChatGPT</a> and how
they can be used to summarise knowledge and answer questions. ChatGPT is sometimes overloaded, but we can try to use it. 
Some users have reported problems getting sensible answers out of it, e.g. <a href="https://twitter.com/itstimconnors/status/1599544717943123969
"  target="_new">@itstimconnors</a>.

<h2><a name="future"/>Questions to think about</h2>

<ul>
<li>How many web sites have machine-readable data available?</li>
<li>How many web sites have an easy way to access their data (e.g., an API, an R package, etc.)?</li>
<li>Do web sites reuse existing identifiers (e.g., for species) or do they have their own identifiers (why would this be a problem?)</li>
<li>What sorts of information are missing or hard to find?</li>
<li>How important is crowd-sourced data (e.g., from sources such as iNaturalist)?</li>
<li>What is the role of machine learning and AI tools such as <a href="https://chat.openai.com">ChatGPT</a>?</li>
</ul>

<h2>Reading</h2>
<ul>
<li>Van Horn G, Mac Aodha O, Song Y, Cui Y, Sun C, Shepard A, Adam H, Perona P, Belongie S (2017) The iNaturalist Species Classification and Detection Dataset. In: arXiv.org. <a href="https://arxiv.org/abs/1707.06642v2">https://arxiv.org/abs/1707.06642v2. Accessed 10 Jul 2023</a></li>
<li>Horn GV, Aodha OM, Song Y, Cui Y, Sun C, Shepard A, Adam H, Perona P, Belongie S (2018) The iNaturalist Species Classification and Detection Dataset. IEEE Computer Society, pp 8769–8778 <a href="https://doi.org/10.1109/CVPR.2018.00914">https://doi.org/10.1109/CVPR.2018.00914</a></li>
<li>Wardeh, M., Risley, C., McIntyre, M. et al. Database of host-pathogen and related species interactions, and their global distribution. Sci Data 2, 150049 (2015). <a href="https://doi.org/10.1038/sdata.2015.49">https://doi.org/10.1038/sdata.2015.49</a></li>

</ul>

<h2>Comments</h2>
<div id="disqus_thread"></div>
<script>
    /**
    *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
    *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
    /*
    var disqus_config = function () {
    this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
    this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
    };
    */
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://teaching-online.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>


</body>
</html>
